### PR TITLE
Rename auction status field to delivery_status

### PR DESCRIPTION
**Pull request by harrisj**
Fixes part of #1247

This is to make more clear how it is different from the bidding status (formerly known as AuctionStatus) and c2_status

### CODE REVIEW

File: app/models/admin_auction_status_presenter_factory.rb [hunk 1]
```diff
@@ -67,11 +67,11 @@ def work_in_progress?
...
-  def status
-    auction.status.camelize
+  def delivery_status
+    auction.delivery_status.camelize
   end
 
   def c2_status
     auction.c2_status.camelize
-  end
+ps  end
```
Comment (jessieay) [inline app/models/admin_auction_status_presenter_factory.rb]: typo?

File: spec/factories/auctions.rb [hunk 2]
```diff
@@ -141,12 +141,12 @@
       end
     end
 
-   trait :pending_c2_approval do
-     future
-     c2_status :pending_approval
-     purchase_card :default
-     unpublished
-   end
+    trait :pending_c2_approval do
```
Comment (jessieay) [inline spec/factories/auctions.rb]: I can't tell if this new indentation is good or bad?

File: app/models/admin_auction_status_presenter_factory.rb (hunk 1 above)
Comment (harrisj) [inline app/models/admin_auction_status_presenter_factory.rb]: Yes fixed

File: spec/factories/auctions.rb (hunk 2 above)
Comment (harrisj) [inline spec/factories/auctions.rb]: Actually, the issue was the indenting was bad before and I fixed it

**Comment by jessieay** [comment]
Perfecto -- mergin'! 😸

**Pull request merged**